### ISSUE TITLE
涂鸦标注数据预处理

### ISSUE THREAD

**Issue by LiuMingYy** [dddraxxx/Weakly-Supervised-Camouflaged-Object-Detection-with-Scribble-Annotations#10]
您好！非常感谢您出色的工作！在拜读您的代码的过程中，对data文件夹下dataset.py中的第110-112行代码，mask[mask == 0.] = 255.   mask[mask == 2.] = 0. ，论文中说0代表未标注，1代表目标部分标注，2代表背景部分标注，请问这里为什么只处理了0和2的情况，而没有对mask == 1.进行处理呢？

**Comment by dddraxxx** [comment]
这只是data预处理阶段，training code里有处理

**Comment by ciwei-01** [comment]
Except for the file paths, the rest of the code doesn’t need to be modified, right? Why does it seem that during training the background is being treated as the target (foreground) region? This causes the model to fail to learn properly and the loss remains very high.
